micropip/_micropip.py

```diff
@@ -89,6 +89,12 @@ def from_url(url: str) -> "WheelInfo":
             parsed_url=parsed_url,
         )
 
+    def best_compatible_tag_index(self) -> int | None:
```

Comment:
Please add a docstring at least with a 1 line description

Reply:
They are all supposed to have docstrings but sometimes I swindle Roman out of them.